nextcord/audit_logs.py

```diff
@@ -462,11 +462,10 @@ def _from_data(self, data: AuditLogEntryPayload) -> None:
 
         # this key is technically not usually present
         self.reason = data.get("reason")
-        self.extra = data.get("options", {})  # type: ignore
-        # I gave up trying to fix this
+        self.extra = extra = data.get("options", {})  # type: ignore
```

Comment:
Is this meant to be changed to `extra = ...`? Otherwise the rest of the code changes don't work

Reply:
Why wouldn't they? This double assignment is kinda a patch for pyright inference constantly changing every update. With this inference and at least one use of it, it works, which is so irritating.

Reply:
Sorry, didn't see the double assign here when reviewing